File: ANGEN/SampleFramework.Controls/SmoothControls - User's Guide/Samples/Notification Category/Fake Minesweeper/CS/FakeMinesweeper/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace FakeMinesweeper
{
	public partial class MainForm : Form
	{
		private void _goButton_Click(object sender, EventArgs e)
		{
			_timer.Enabled = true;
		}

		private void _timer_Tick(object sender, EventArgs e)
		{
			int hundreds = _counter / 100;
			int tempNum = _counter - hundreds * 100;
			int tens = tempNum / 10;
			tempNum = tempNum - tens * 10;

			_segmentOne.Value = hundreds;
			_segmentTwo.Value = tens;
			_segmentThree.Value = tempNum;

			_counter++;

			if (_counter == 1000)
			{
				_counter = 0;
			}
		}

		private int _counter;

		public MainForm()
		{
			InitializeComponent();
		}
	}
}

File: ANGEN/SampleFramework.Controls/SmoothControls - User's Guide/Samples/Notification Category/Fake Minesweeper/VB/FakeMinesweeper/MainForm.vb
Public Class MainForm
    Private Sub _timer_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles _timer.Tick
        Dim hundreds As Int32 = _counter / 100
        Dim tempNum As Int32 = _counter - hundreds * 100
        Dim tens As Int32 = tempNum / 10
        tempNum = tempNum - tens * 10

        _segmentOne.Value = hundreds
        _segmentTwo.Value = tens
        _segmentThree.Value = tempNum

        _counter += 1

        If _counter = 1000 Then
            _counter = 0
        End If
    End Sub

    Private Sub _goButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles _goButton.Click
        _timer.Enabled = True
    End Sub

    Dim _counter As Int32

    Sub New()
        Me.InitializeComponent()
    End Sub
End Class
